yarn-project/merkle-tree/src/snapshots/append_only_snapshot.ts

```diff
@@ -0,0 +1,179 @@
+import { Hasher, SiblingPath } from '@aztec/types';
+
+import { LevelUp } from 'levelup';
+
+import { AppendOnlyTree } from '../interfaces/append_only_tree.js';
+import { SiblingPathSource } from '../interfaces/merkle_tree.js';
+import { TreeBase } from '../tree_base.js';
+import { SnapshotBuilder } from './snapshot_builder.js';
+
+const nodeVersionKey = (name: string, level: number, index: bigint) =>
```

Comment:
Would be neat with a one line comment for each of these keys for what is to be stored at them.